### DISCUSSION TITLE
Upgrading Clarity and Angular in tandem

### DISCUSSION THREAD

**Discussion by mindphreak** [vmware-clarity/ng-clarity#372] [Q&A] [answered]
I'm on Angular 11 and Clarity  4 at the moment. I'm upgrading Angular to 14 step by step. How do I upgrade clarity in tandem so that it will work with each version of Angular during the process?

**Comment by kevinbuhmann** [comment, marked as answer]
- Update to Angular v12 and @clr/* v12.
- Update to Angular v13 and @clr/* v13.
- Update to Angular v14 and stay on @clr/* v13 (no v14).
